src/providers/twitch/api/Helix.hpp

```diff
@@ -329,6 +330,16 @@ struct HelixChatSettings {
     }
 };
 
+class HelixUserList {
+    public:
+        std::unordered_set<QString> users;
+        int total;
+
+        HelixUserList();
+
+        void AddUsersFromResponse(const QJsonObject &response);
```

Comment:
Casing should be `addUsersFromResponse`

Preferably the function is protected/private with only the Helix class having access to it so users don't get the idea that it can be called

Reply:
Class deleted